### PR TITLE
Fai 1956 unsubscribesavedsearch

### PR DESCRIPTION
**Pull request by LACrossley**

### CODE REVIEW

File: src/FindAnApprenticeship/SFA.DAS.FindAnApprenticeship/Application/Commands/SaveSearch/UnsubscribeSavedSearchCommand.cs [hunk 1]
```diff
@@ -0,0 +1,11 @@
+using System;
+using MediatR;
+
+namespace SFA.DAS.FindAnApprenticeship.Application.Commands.SaveSearch;
+
+public record UnsubscribeSavedSearchCommand(Guid Id) : IRequest;
+
+//public class UnsubscribeSavedSearchCommand : IRequest
+//{
```
Comment (balaji-j) [inline src/FindAnApprenticeship/SFA.DAS.FindAnApprenticeship/Application/Commands/SaveSearch/UnsubscribeSavedSearchCommand.cs:9]: Please could you remove the un-used code or commented ones

File: src/FindAnApprenticeship/SFA.DAS.FindAnApprenticeship.Api.UnitTests/Controllers/SavedSearchController/WhenGettingUnsubscribe.cs [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+        [Frozen] Mock<IMediator> mediator,
+        [Greedy] Api.Controllers.SavedSearchesController savedSearchController)
+    {
+        mediator.Setup(x => x.Send(It.Is<GetUnsubscribeSavedSearchQuery>(c => c.SavedSearchId == id),
+            It.IsAny<CancellationToken>())).ReturnsAsync(queryResult);
+
+        var actual = await savedSearchController.GetUnsubscribeSavedSearch(id) as OkObjectResult;
+
+        actual.Should().NotBeNull();
+        actual!.Value.Should().Be(queryResult);
```
Comment (balaji-j) [inline src/FindAnApprenticeship/SFA.DAS.FindAnApprenticeship.Api.UnitTests/Controllers/SavedSearchController/WhenGettingUnsubscribe.cs:29]: Please use BeEquivalentTo to compare the properties as well

File: src/FindAnApprenticeship/SFA.DAS.FindAnApprenticeship/Application/Commands/SaveSearch/UnsubscribeSavedSearchCommand.cs (hunk 1 above)
Comment (LACrossley) [reply src/FindAnApprenticeship/SFA.DAS.FindAnApprenticeship/Application/Commands/SaveSearch/UnsubscribeSavedSearchCommand.cs:9]: Addressed in latest commit.

File: src/FindAnApprenticeship/SFA.DAS.FindAnApprenticeship.Api.UnitTests/Controllers/SavedSearchController/WhenGettingUnsubscribe.cs (hunk 2 above)
Comment (LACrossley) [reply src/FindAnApprenticeship/SFA.DAS.FindAnApprenticeship.Api.UnitTests/Controllers/SavedSearchController/WhenGettingUnsubscribe.cs:29]: Addressed in latest commit.

**Pull request merged**